tests/cli/test_cli.py

```diff
@@ -280,7 +280,9 @@ def test_kyc_manage(
         ]
     )
     assert result.exit_code == 0
-    assert kyc_contract.functions.isWhitelisted(eth_address).call() == True
+    # TODO: this method is no longer supported but maybe this should be tested
```

Comment:
This has changed in the new abi. You should check the `getAttribute` method in the new ABI. cc @villesundell .